src/core/createSaleorClient.ts

```diff
@@ -11,6 +11,7 @@ export const createSaleorClient = ({
   apiUrl,
   channel,
   autologin = true,
+  fetchOptions,
```

Comment:
Merge `autologin` with `fetchOptions`:
```
apiUrl,
chanel,
opts = { autologin = true, fetchOpts = default }
```

Reply:
Default fetch options are in createFetch method